src/training/baseline.py

```diff
@@ -29,19 +31,17 @@ def __init__(self,
         self.total_epochs = total_epochs
         self.logger = logger
         self.storage = storage
-        self.ocr_loss = ocr.OCRLoss().to(device)
+        self.ocr_loss: nn.Module = OCRLoss().to(device)
```

Comment:
Well... If you decided to restrict this piece of code too, it might be better to pass loss function from config and mark variable type in function's args.